### PR TITLE
feat: [M3-7366] - Improved Primary Navigation UX

### PR DESCRIPTION
**Pull request by jaalah-akamai**
## Description 📝
Overall improvements to primary nav, which includes changes to accommodate longer Cloud Manager feature names.

## Changes  🔄
- Increased nav width to 232px
- Added transition to nav links and adjusted logo transition to match
- Tightened spacing around logos, links, and beta chip according to mocks
- Reduced link font size to .875rem/14px

## Screenshot
| Before      | After |
| ----------- | ----------- |
|Screenshot 2024-02-07 at 12 35 54 PM|Screenshot 2024-02-07 at 12 37 02 PM|

## How to test 🧪
- Checkout branch or preview link
- Verify that opening / closing menu works as intended
- Verify that changing breakpoints works as intended
- Verify changes in different browsers

### CODE REVIEW

File: packages/manager/public/fonts/fonts.css [hunk 1]
```diff
@@ -11,6 +11,19 @@
...
+  src: url('fonts/Lato-Semibold.eot'); /* IE9 Compat Modes */
+  src: url('fonts/Lato-Semibold.eot?#iefix') format('embedded-opentype'),
+    /* IE6-IE8 */ url('fonts/Lato-Semibold.woff2') format('woff2'),
+    /* Modern Browsers */ url('fonts/Lato-Semibold.woff') format('woff'),
+    /* Modern Browsers */ url('fonts/Lato-Semibold.ttf') format('truetype');
+  font-style: normal;
+  font-weight: normal;
+  text-rendering: optimizeLegibility;
+}
+
```
Comment (abailly-akamai) [inline packages/manager/public/fonts/fonts.css:26]: I had removed this font cause it was used in maybe two places and I'd hate to see it resurface. This is KiloBytes of initial download we can avoid by using the regular light font (which the mockups actually call for)

File: packages/manager/src/MainContent.tsx [hunk 2]
```diff
@@ -74,7 +74,7 @@ const useStyles = makeStyles()((theme: Theme) => ({
   content: {
     flex: 1,
     [theme.breakpoints.up('md')]: {
-      marginLeft: 190,
+      marginLeft: 232,
```
Comment (abailly-akamai) [inline packages/manager/src/MainContent.tsx:77]: mockups called for 245 i think?

Comment (jaalah-akamai) [reply packages/manager/src/MainContent.tsx:77]: We made a last minute change to 232 to match CDS - I'll have Kendall update mocks

File: packages/manager/public/fonts/fonts.css (hunk 1 above)
Comment (jaalah-akamai) [reply packages/manager/public/fonts/fonts.css:26]: They determined it should be bolder, what if we did regular with a heavier font-weight like 600? I think that could work as a compromise.

Comment (abailly-akamai) [reply packages/manager/public/fonts/fonts.css:26]: where is this all being captured? we had an initial ticket for this

**Comment by jaalah-akamai** [comment]
@abailly-akamai Got the green light from Product/UX. So opening this for review

File: packages/manager/src/MainContent.tsx (hunk 2 above)
Comment (bnussman-akamai) [inline packages/manager/src/MainContent.tsx:77]: Would using the `SIDEBAR_WIDTH` value make sense here?

Comment (abailly-akamai) [reply packages/manager/src/MainContent.tsx:78]: Agreed it would

**Review by abailly-akamai — COMMENTED** [review]
@jaalah-akamai heads up i rebased the PR

I think we still need to handle the vertical centering of both the logo and the managed menu item?

Screenshot 2024-03-13 at 10 20 22

**Comment by jaalah-akamai** [comment]
| Managed | Default |
| ------- | ------- |
|Screenshot 2024-03-14 at 1 16 37 PM|Screenshot 2024-03-14 at 1 18 09 PM|

**Review by bnussman-akamai — COMMENTED** [review]
Nice! Looks slightly better imo!

I do feel like the green accent colors need to go 🗑️ Is this something we'd consider doing now?

For example, blue looks slightly more appealing to me
Screenshot 2024-03-14 at 1 43 24 PM

**Comment by abailly-akamai** [comment]
@bnussman-akamai that's a great suggestion - let's go blue! @jaalah-akamai?

**Comment by mjac0bs** [comment]
> @bnussman-akamai that's a great suggestion - let's go blue! @jaalah-akamai?

goblue


As an alum, I couldn't resist this one opportunity. 😆

**Comment by jaalah-akamai** [comment]
@bnussman-akamai @abailly-akamai @mjac0bs UX wants to hold on the color change especially since we'll be introducing design tokens soon and the colors will shift slightly. It's on our radar though!

**Comment by jaalah-akamai** [comment]
This is good to go otherwise

File: packages/manager/src/assets/logo/akamai-logo.svg
Comment (bnussman-akamai) [inline packages/manager/src/assets/logo/akamai-logo.svg:1]: Can we retain the colored logo for Storybook? I think the colored logo looked nice there!

Screenshot 2024-03-27 at 11 40 53 AM

File: packages/manager/.changeset/pr-10137-tech-stories-1707328190628.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+"@linode/manager": Tech Stories
+---
+
+Improved Primary Navigation UX ([#10137](https://github.com/linode/manager/pull/10137))
```
Comment (mjac0bs) [inline packages/manager/.changeset/pr-10137-tech-stories-1707328190628.md:5]:
```suggestion
Improve Primary Navigation UX ([#10137](https://github.com/linode/manager/pull/10137))
```
Nit: present tense

**Review by mjac0bs — APPROVED** [review]
This looks great on Chrome, Safari, and Firefox at various breakpoints. 🚀 

The only thought I had (and perhaps only because I was sitting there clicking this button repeatedly), is that the fade out of the text feels a little slow when collapsing the menu.

https://github.com/linode/manager/assets/114685994/272fa36c-d0aa-4c40-906d-8229619acf68

**Review by hana-linode — APPROVED** [review]
I feel like the divider spacing between the logo and the items are a bit cramped in comparison to the other dividers in the nav. Can we increase the spacing? LGTM otherwise!

image

**Review by bnussman-akamai — APPROVED** [review]

**Comment by jaalah-akamai** [comment]
> I feel like the divider spacing between the logo and the items are a bit cramped in comparison to the other dividers in the nav. Can we increase the spacing? LGTM otherwise!

That's by design so that the border lines up with the top menu

**Pull request merged**